jina/docker/hub_test_generators.py

```diff
@@ -0,0 +1,29 @@
+import random
+import string
+
+import numpy as np
+
+from jina import Document
+
+
+def text_document_generator(num_docs):
+    for i in range(num_docs):
+        length = random.randint()
+        document = Document(text=''.join(random.choices(string.ascii_uppercase + string.digits, k=length)))
+        yield document
+
+
+def image_document_generator(num_docs):
+    length = random.randint()
+    height = random.randint()
```

Comment:
They should be parameters and only random if not set. Imagine someone does
```
index_docs = image_document_generator(10)
index index_docs...
query_docs = image_document_generator(5)
query query_docs
```
Then they would not have equal size which might crash out of the box.

Reply:
Do not look at the details now. this is very very early stage and far from being ready. this will not be used by users outside though